### DISCUSSION TITLE
Support `websearch_to_tsvector` in `full_text_search`

### DISCUSSION THREAD

**Discussion by rgalanakis** [jeremyevans/sequel#2075] [General]
On this line: https://github.com/jeremyevans/sequel/blob/master/lib/sequel/adapters/shared/postgres.rb#L1819

Would be nice to be able to use `:websearch` too (which is `websearch_to_tsquery` vs. `plainto_tsquery` so isn't just adding another option.

Also maybe consider using the `to_tsquery` value verbatim if it ends with `"to_tsquery"`? That way it'd be proofed against new tsquery routines.

In the meantime I know I can pass in my own expression but it's not as clean. Thanks!

**Comment by jeremyevans** [comment]
I'm open to added support for that.  Can you send in a pull request with adapter tests for it?

**Reply by jeremyevans** [reply]
I added support for this in d74dc5c89a5190bcddcf34f2ff0ecbe0e6969e30

**Comment by rgalanakis** [comment]
You're a real mensch! Thanks Jeremy and sorry I wasn't able to get to it.
